aries-site/src/data/wcag/components.json

```diff
@@ -12,6 +12,24 @@
       { "rule": "2.4.9", "status": "conditional" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
+      { "rule": "1.4.3", "status": "passed" },
+      { "rule": "2.1.1", "status": "passed" },
+      { "rule": "2.1.2", "status": "passed" },
+      { "rule": "2.4.7", "status": "passed" },
+      { "rule": "2.4.11", "status": "passed" },
+      { "rule": "2.4.12", "status": "passed" },
+      { "rule": "2.4.13", "status": "passed" },
```

Comment:
I think this rule is failing

Reply:
I was also on the fence about this one.

```
The first part of the Success Criterion specifies a minimum area for the focus indicator:

is at least as large as the area of a 2 [CSS pixel](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html#dfn-css-pixel) thick [perimeter](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html#dfn-perimeter) of the unfocused component or sub-component
This only specifies a minimum area for the focus indicator. It does not require that the focus indicator literally be a 2 CSS pixel thick outline, only that the indicator be at least that large.

However, the simplest way to meet the size requirement is to use a focus indicator which is a solid 2 CSS pixel thick perimeter.
```

Reply:
When reading this rule I took it that this rule was to ensure that there is a clear visual indicator for users when the element is in focus. Our focus color does have sufficient contrast and the `box-shadow` itself is 2px. So I would say that our box shadow is visually distinct enough to serve as a clear indicator that the Menu is being focused.